lib/block-supports/typography.php

```diff
@@ -461,7 +461,6 @@ function gutenberg_get_typography_font_size_value( $preset, $should_use_fluid_ty
 	$default_maximum_viewport_width   = '1600px';
 	$default_minimum_viewport_width   = '768px';
 	$default_minimum_font_size_factor = 0.75;
-	$default_maximum_font_size_factor = 1.5;
```

Comment:
Removed since we never need to compute the max size that we fallback to the incoming "size" or single/custom value. See issue: #45504